src/components/Subteams/subteams/OutreachTeam.jsx

```diff
@@ -11,7 +11,7 @@ function OutreachTeam() {
                 </div>
                 <div className="left">
                     <h1 className="captain-title">Outreach</h1>
-                    <p className="sub-paragraph">Outreach is one of the great opportunities available on the team. Outreach has an overall job to use our robotics skills to better the community. We solder easy-press buttons onto toys in order to make them more inclusive because some kids might not be able to play with them otherwise. Additionally, we create and design a kickball device called CHUCK (Community-Helping Utility Companion for Kickball). This device allows every kid, regardless of their mobility, to participate in games.
+                    <p className="sub-paragraph">Outreach is one of the great opportunities available on the team. Outreach has an overall job to use our robotics skills to better the community. As part of this program we create Chucks and Easy-press buttons, both of which help children with mobility restrictions engage in new oppurtinties.
```

Comment:
I think you can summarize this a bit - Outreach is one of the great opportunities available on the team, using our robotics skills to better the community. Through this program, we create CHUCKs and Easy-Press buttons, which help children with mobility restrictions engage in new opportunities.